### PR TITLE
Update captions for ruskj file

### PR DESCRIPTION
**Pull request by ruskj**
Related to #2

### CODE REVIEW

File: _posts/0000-01-07-ruskj.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-hide](https://cloud.githubusercontent.com/assets/16547949/25400902/0a553f14-29c2-11e7-9a40-8d14136b4fa3.jpg)
 {: .center}
 
-Caption here
+Doh!
```
Comment (maricay) [inline _posts/0000-01-07-ruskj.md]: reviewing changes

Comment (maricay) [reply _posts/0000-01-07-ruskj.md]: reviewing changes 2

Comment (maricay) [reply _posts/0000-01-07-ruskj.md]: reviewing changes 3

Comment (maricay) [reply _posts/0000-01-07-ruskj.md]: END

Comment (burnsa2018) [reply _posts/0000-01-07-ruskj.md]: This code is really sad.

**Pull request merged**